Wdvh/StarWarsTitleOnly.py

```diff
@@ -0,0 +1,270 @@
+from pathlib import Path
+from re import match
+
+from num2words import num2words
+
+from modules.CardType import CardType
+from modules.Debug import log
+
+class StarWarsTitleOnly (CardType):
+    """
+    This class describes a type of ImageMaker that produces title cards in the
+    theme of Star Wars cards as designed by reddit user /u/Olivier_286. These
+    cards are not as customizable as the standard template.
+    """
+
+    """Directory where all reference files used by this card are stored"""
+    REF_DIRECTORY = Path(__file__).parent / 'ref' / 'star_wars'
```

Comment:
```suggestion
    REF_DIRECTORY = Path(__file__).parent.parent / 'ref' / 'star_wars'
```
See [here](https://github.com/CollinHeist/TitleCardMaker-CardTypes/wiki/Creating-a-Custom-Card-Type#reference-directory)